src/praisonai/tests/unit/test_autogen_backward_compatibility.py

```
@@ -0,0 +1,466 @@
+"""
+AutoGen Backward Compatibility Tests
+
+This test module ensures that the introduction of AutoGen v0.4 support
+maintains full backward compatibility with existing v0.2 implementations.
+"""
+
+import pytest
+import os
+import sys
+from unittest.mock import Mock, MagicMock, patch
```

Ruff (0.12.2)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)